db/sql/initialization/extrinsic.sql

```diff
@@ -8,7 +8,8 @@ CREATE TABLE IF NOT EXISTS extrinsic (
   index BIGINT NOT NULL,
   hash TEXT NOT NULL,
   args JSON NOT NULL,
-  docs TEXT NOT NULL, 
+  contract_address_evm_call VARCHAR(48),
```

Comment:
In my opinion, this does not belong here...
EVM addresses are present in events. 

Let's do it like we discussed to have one separate table for EVM events and have foreign keys on contract and event